render/meson.build

```diff
@@ -1,13 +1,33 @@
+renderers = get_option('renderers')
+egl = dependency('egl', required : false)
+gbm = dependency('gbm', version: '>=17.1.0', required : false)
```

Comment:
Like I said below, I'd prefer to always require GBM for now, and make it optional later, once we sort out the issues with the DRM backend.

Reply:
That is right, I thought we use the surface from gbm, that is what weston does. But wlroots didn't use gbm surface or EGL surface, it would also using EGLImage -> Renderbuffer -> FBO
I would move egl and gles2 into the same directory and EGL doesn't depend on gbm.